src/SoulMemory/DarkSouls1/KnownFlag.cs

```diff
@@ -0,0 +1,182 @@
+
+using System.Xml.Serialization;
+
+namespace SoulMemory.DarkSouls1
+{
+    [XmlType(Namespace = "SoulMemory.DarkSouls1")]
+    public enum KnownFlag : uint
+    {
+
+      [AnnotationAttribute(Name = "Bell of Awakening, Undead Parish 11010700", Description = "Bells of Awakening")]
+      BellOfAwakeningUndeadParish = 11010700,
+
+      [AnnotationAttribute(Name = "Bell of Awakening, Blighttown 11400200", Description = "Bells of Awakening")]
+      BellOfAwakeningBlighttown = 11400200,
+
+      [AnnotationAttribute(Name = "Other, Dark Anor Londo 11510400", Description = "Lordvessel")]
+      OtherDarkAnorLondo = 11510400,
+
+      [AnnotationAttribute(Name = "Other, Receive Lordvessel With Gwynevere Still Alive 11510592", Description = "Lordvessel")]
+      OtherReceiveLordvesselWithGwynevereStillAlive = 11510592,
+
+      [AnnotationAttribute(Name = "Other, Receive Lordvessel (Gwynevere Alive or Dead) 50000090", Description = "Lordvessel")]
+      OtherReceiveLordvesselGwynevereAliveOrDead = 50000090,
+
+      [AnnotationAttribute(Name = "Anor Londo, Door to Gwynevere 11510110", Description = "Doors")]
+      AnorLondoDoorToGwynevere = 11510110,
+
+      [AnnotationAttribute(Name = "Darkroot Garden, Crest of Artorias Door 61200500", Description = "Doors")]
+      DarkrootGardenCrestOfArtoriasDoor = 61200500,
+
+      [AnnotationAttribute(Name = "Darkroot Garden, Sif Door 61200501", Description = "Doors")]
+      DarkrootGardenSifDoor = 61200501,
+
+      [AnnotationAttribute(Name = "Demon Ruins, Shortcut Door to Lost Izalith 11410340", Description = "Doors")]
+      DemonRuinsShortcutDoorToLostIzalith = 11410340,
+
+      [AnnotationAttribute(Name = "Northern Undead Asylum, Cell Door 11810103", Description = "Doors")]
+      NorthernUndeadAsylumCellDoor = 11810103,
+
+      [AnnotationAttribute(Name = "Northern Undead Asylum, Asylum Demon Door 11810112", Description = "Doors")]
+      NorthernUndeadAsylumAsylumDemonDoor = 11810112,
+
+      [AnnotationAttribute(Name = "Northern Undead Asylum, Big Pilgrim Door 11810110", Description = "Doors")]
+      NorthernUndeadAsylumBigPilgrimDoor = 11810110,
+
+      [AnnotationAttribute(Name = "Anor Londo, Elevator Active 11510220", Description = "Elevators")]
+      AnorLondoElevatorActive = 11510220,
+
+      [AnnotationAttribute(Name = "Catacombs, Entrance Lever 11300900", Description = "Levers")]
+      CatacombsEntranceLever = 11300900,
+
+      [AnnotationAttribute(Name = "Duke's Archives, First Lever 11705091", Description = "Levers")]
+      DukesArchivesFirstLever = 11705091,
+
+      [AnnotationAttribute(Name = "Duke's Archives, Second Lever 11705090", Description = "Levers")]
+      DukesArchivesSecondLever = 11705090,
+
+      [AnnotationAttribute(Name = "New Londo Ruins, Seal Opened 11600200", Description = "Levers")]
+      NewLondoRuinsSealOpened = 11600200,
+
+      [AnnotationAttribute(Name = "NPC, Andre, Talk To For First Time 71010000", Description = "NPC")]
```

Comment:
Is "NPC" duplicate here? Maybe only the description for grouping as NPC is enough?

Reply:
I think it's useful for people looking for NPC flags if they can't remember their name